assets/src/edit-story/app/api/apiProvider.js

```diff
@@ -386,6 +387,34 @@ function APIProvider({ children }) {
     [cdnURL, assetsURL]
   );
 
+  // @todo Add paging.
+  const getCustomPageTemplates = useCallback(() => {
+    let apiPath = customPageTemplates;
+    const perPage = 100;
+    apiPath = addQueryArgs(apiPath, {
+      context: 'edit',
+      per_page: perPage,
+      page: 1,
+    });
+    return apiFetch({ path: apiPath }).then((response) =>
+      response.map((template) => template['story_data'])
```

Comment:
Instead of a map here, could we use a array pop or shift to get the first only item returned.

Reply:
But we need all the items, this is getting all the custom templates.